src/app/workflow-form/workflow-form.component.ts

```diff
@@ -83,21 +86,67 @@ export class WorkflowFormComponent implements OnInit, OnChanges {
   locked = true;
   taskStates = WorkflowTaskState;
 
+  documentationTitles = Array<string>();
+  documentationBlocks = Array<string>();
+
   constructor(
     private api: ApiService,
     private location: Location,
+    public dialog: MatDialog,
   ) {
   }
 
   ngOnInit() {
     this._loadModel(this.task);
+    this.processElementDocumentation(this.task.documentation);
   }
 
   ngOnChanges(changes: SimpleChanges): void {
     if (changes.task && changes.task.currentValue) {
       this._loadModel(changes.task.currentValue);
     }
   }
+  openDialog(markdown: string) {
+    this.dialog.open(WorkflowDialogComponent, {
+      data: markdown
+    });
+  }
+
+  // processElementDocumentation(markdown: string) {
+  //   // const reg = /(<\s*info[^>]*>)([\s\S]*?)<\s*\/\s*info>/g;
+  //   const data = markdown.split("<info>");
+  //   const main = data[0];
+  //   const info = data[1];
+
+  //   this.documentationTitles = [main];
+  //   this.documentationBlocks = [info];
+
+  // }
+  processElementDocumentation(markdown: string) {
+    const reg = /(?:<\s*block(?:\s+?id=["'](.+?)["'])?[^>]*>)([\s\S]*?)(?:<\s*\/\s*block>)/g;
+    const titles = [];
+    const helpTexts = [];
+    const matches = [...markdown.matchAll(reg)];
```

Comment:
in TypeScript, a string does not have a matchAll function on it.   Are you sure this is workign as expected, my editor is complaining.